developer/tryout/samples/filters/sample-custom-filter/src/main/java/org/example/tests/CustomFilter.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package org.example.tests;
+
+import org.wso2.apk.enforcer.commons.model.APIConfig;
+import org.wso2.apk.enforcer.commons.model.RequestContext;
+import org.wso2.apk.enforcer.commons.Filter;
+
+import java.util.Map;
+
+public class CustomFilter implements Filter {
+    private Map<String, String> configProperties;
+
+    @Override
+    public void init(APIConfig apiConfig, Map<String, String> configProperties) {
+        this.configProperties = configProperties;
+    }
+
+    @Override
+    public boolean handleRequest(RequestContext requestContext) {
+        String headerValue = configProperties.get("CustomProperty");
+        requestContext.addOrModifyHeaders("Custom-header-1", headerValue);
```

Comment:
shall we add some other capabilities too?
- add/remove query param
- direct response

Reply:
Removed the sample project from 27220cb7252061a0dae73d7205c44f1fbe6850c3,  since this functionality is not used anymore